rllib/core/models/tests/test_catalog.py

```diff
@@ -0,0 +1,150 @@
+import unittest
+
+import gym
+import numpy as np
+import itertools
+
+from gymnasium.spaces import Box
+
+from ray.rllib.core.models.catalog import Catalog
+from ray.rllib.core.models.configs import MLPEncoderConfig, CNNEncoderConfig
+
+
+class TestCatalog(unittest.TestCase):
+    def test_get_encoder_config(self):
+        """Tests if we can create a bunch of encoders from the base catalog class."""
+
+        # TODO (Artur): Add support for the commented out spaces
+        input_spaces_and_config_types = [
+            (Box(-1.0, 1.0, (5,), dtype=np.float32), MLPEncoderConfig),
+            (Box(-1.0, 1.0, (84, 84, 1), dtype=np.float32), CNNEncoderConfig),
+            (Box(-1.0, 1.0, (240, 320, 3), dtype=np.float32), CNNEncoderConfig),
```

Comment:
I'm testing CNN stuff here already, but it will only make it to other tests in an upcoming PR